src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -538,19 +456,13 @@ private bool ValidateNonce(ContractEntry entry, ExecutionEnvironment env, ITxTra
         /// <exception cref="TransactionCollisionException">Thrown when the address is already in use</exception>
         private void PrepareContractAccount(Address recipient)
         {
-            if (!_stateProvider.AccountExists(recipient))
-            {
-                return;
-            }
-            
+            if (!_stateProvider.AccountExists(recipient)) return;
+
             var addressHasCode = (_virtualMachine.GetCachedCodeInfo(recipient)?.MachineCode?.Length ?? 0) != 0;
             var addressWasUsed = _stateProvider.GetNonce(recipient) != 0;
             if (addressHasCode || addressWasUsed)
             {
-                if (_logger.IsEnabled(LogEventLevel.Verbose))
-                {
-                    _logger.Verbose($"Contract collision at {recipient}");
-                }
+                if (_logger.IsEnabled(LogEventLevel.Verbose)) _logger.Verbose($"Contract collision at {recipient}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4614986)